internal/blocksync/v0/reactor.go

```diff
@@ -299,8 +299,8 @@ func (r *Reactor) handleMessage(chID p2p.ChannelID, envelope p2p.Envelope) (err
 }
 
 // processBlockchainCh initiates a blocking process where we listen for and handle
-// envelopes on the BlockchainChannel and blockchainOutBridgeCh. Any error encountered during
-// message execution will result in a PeerError being sent on the BlockchainChannel.
+// envelopes on the BlockSyncChannel and blockchainOutBridgeCh. Any error encountered during
```

Comment:
What's blockchainOutBridgeCh? Is that an actual channel

Reply:
Not sure, It was there, so kept it. Can remove it though, if needed.

Reply:
Oh nvm, I've updated the `blockchainOutBridgeCh` to `blockSyncOutBridgeCh`